.github/workflows/tests.yml

```
@@ -11,3 +11,48 @@ jobs:
       php-versions: '["8.2", "8.3", "8.4"]'
       typo3-versions: '["12.4", "13.4"]'
       dependencies: '["highest", "lowest"]'
+
+  tests-acceptance:
+    name: Acceptance testing TYPO3 ${{ matrix.typo3-version }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        typo3-versions: '["12", "13"]'
```

actionlint (1.7.7)

expecting a single ${{...}} expression or array value for matrix variations, but found plain text node

(syntax-check)